lsp-sqls.el

```diff
@@ -0,0 +1,67 @@
+;;; lsp-sqls.el --- SQL Client settings -*- lexical-binding: t; -*-
+
+;; Copyright (C) 2020 Shunya Ishii
+
+;; Author: Shunya Ishii
+;; Keywords: sql lsp
+
+;; This program is free software; you can redistribute it and/or modify
+;; it under the terms of the GNU General Public License as published by
+;; the Free Software Foundation, either version 3 of the License, or
+;; (at your option) any later version.
+
+;; This program is distributed in the hope that it will be useful,
+;; but WITHOUT ANY WARRANTY; without even the implied warranty of
+;; MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+;; GNU General Public License for more details.
+
+;; You should have received a copy of the GNU General Public License
+;; along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+;;; Commentary:
+
+;; LSP client for SQL
+
+;;; Code:
+
+(require 'lsp-mode)
+(require 'json)
+
+(defgroup lsp-sqls nil
+  "LSP support for SQL, using sqls"
+  :group 'lsp-mode
+  :link '(url-link "https://github.com/lighttiger2505/sqls")
+  :package-version `(lsp-mode . "6.3"))
+
+(defcustom lsp-sqls-server "sqls"
+  "Path to the `sqls` binary."
+  :group 'lsp-sqls
+  :risky t
+  :type 'file
+  :package-version `(lsp-mode . "6.3"))
+
+(defun lsp-sqls--make-launch-cmd ()
+  (-let [base `(,lsp-sqls-server)]
+    ;; we can add some options to command. (e.g. "-config")
+    base))
+
+(defun lsp-sqls-workspace-config-json ()
+  "Workspace configuration json at workspace root directory."
+  (let ((config-json-path (-> (lsp-workspace-root)
+                              (concat "/.sqls/config.json"))))
+    (if (file-exists-p config-json-path)
+        (json-read-file config-json-path)
+      "{ \"sqls\": {} }")))
```

Comment:
Shouldn't this a JSON object instead of string?
Also `json-read-file` doesn't benefit from native json. Perhaps you can create a `lsp--read-json-file` that looks like `json-read-file` and use `lsp--read-json` to parse json file instead?